src/forms/medication-dispense-review.component.tsx

```diff
@@ -489,6 +500,52 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
             });
           }}
         />
+
+        <OpenmrsDatePicker
```

Comment:
@denniskigen  @ibacher  @pirupius  would be great to get your thoughts on whether I'm using this date component correctly, specifically when setting and updating the date.

One weird thing I'm seeing is this display in the datepicker (12:00AM), which I'm guessing happens because I'm passing a full date (with time) in?

![image](https://github.com/user-attachments/assets/ceba0bce-b9e1-4aa3-b492-11e30060a78a)

And also, do we have control of the date display format, or is this controller by the locale?  We try to avoid the American Month-Day-Year, but maybe it is just displaying that because I'm in the US?

![image](https://github.com/user-attachments/assets/993cefb6-65fb-4ee3-949d-f183b54faad1)

Reply:
Let's start a separate ticket / conversation about this @mogoodrich

Reply:
Yes @mogoodrich the format is based on locale

Reply:
Thanks @pirupius s 

@pirupius @mseaton  I couldn't reproduce this behavior ("Oct 12:00AM") today, I'm wondering if it had anything to do with the fact that it was the first of the month when I was testing?

I did remove the "startOf('day')" as you suggested @pirupius ... I believe that was only something I added in hopes it might fix the above issue.

But I also agree with @mseaton , we should likely start another ticket/convo about the OpenmrsDataPicker and not have it block this ticket.